diff -Naur a/libretro/Makefile b/libretro/Makefile
--- a/libretro/Makefile	2016-01-05 20:18:40.000000000 +0100
+++ b/libretro/Makefile	2016-01-07 08:53:36.356553372 +0100
@@ -18,8 +18,8 @@
 	endif
 endif
 
-CXX			= g++
-CC			= gcc
+CXX             ?= g++
+CC		?= gcc
 TARGET_NAME	= snes9x
 LIBM		= -lm
 
@@ -157,8 +157,8 @@
 	TARGET := $(TARGET_NAME)_libretro.so
 	SHARED := -shared -Wl,--no-undefined
 	fpic := -fPIC
-	CC = gcc
-	CXX = g++
+	CC ?= gcc
+	CXX ?= g++
 	ifneq (,$(findstring cortexa8,$(platform)))
 		CXXFLAGS += -marm -mcpu=cortex-a8
 	else ifneq (,$(findstring cortexa9,$(platform)))
